.gitlab-ci.yml

```diff
@@ -16,11 +16,7 @@ variables:
   CARGO_HOME:                      "/ci-cache/${CI_PROJECT_NAME}/cargo/${CI_COMMIT_REF_NAME}/${CI_JOB_NAME}"
   CARGO_TARGET_DIR:                "/ci-cache/${CI_PROJECT_NAME}/targets/${CI_COMMIT_REF_NAME}/${CI_JOB_NAME}"
   RUST_LIB_BACKTRACE:              "0"
-
-# Necessary for building binaryen-sys, which is part of the binaryen dependency.
-# Should be reverted once the `cargo-contract` ci image provides a clang version
-# for which building `binaryen-sys` works (paritytech/scripts/#237).
-  CXX:                             "/usr/bin/clang++-8"
+  CI_IMAGE:                        "paritytech/contracts-ci-linux:production"
```

Comment:
Just to make sure I understand, if the `contracts-ci-linux` image changes, this CI is triggered but with the above variable pointing at `staging`. If it succeeds the new image will be published as `production`, if it fails we will get a :x: here?

Would be good to get a comment on this side which documents what is going on for future maintainers.

Reply:
Yeah, you'll get a failed pipeline here https://gitlab.parity.io/parity/cargo-contract/-/pipelines?page=1&scope=all&ref=master

HDYT is it enough tho link this PR in the comment?

Reply:
done